src/app/generic-listing-page/generic-listing-page-routing.module.ts

```diff
@@ -28,6 +28,10 @@ const routes: Routes = [
   {
     path: 'spotlight',
     component: SpotlightListingPage
+  },
+  {
+    path:'my-journeys',
+    loadChildren: () => import('../my-journeys-listing/my-journeys-listing.module').then( m => m.MyJourneysListingPageModule)
```

Comment:
Instead of loading the modules
Cant we declare all the components here itself?

Reply:
for the path they are pages but this one is module so loading

Reply:
done